plugin/src/test/groovy/org/openbakery/carthage/AbstractCarthageTaskBaseSpecification.groovy

```diff
@@ -135,6 +136,7 @@ class AbstractCarthageTaskBaseSpecification extends Specification {
 		environment["XCODE_XCCONFIG_FILE"] == xcconfigPath.absolutePath
 	}
 
+	@Ignore
```

Comment:
I think these tests require some sort of specific environment which is problematic. 
@renep Would be great if you have some feedback here or feel free to push an update here.

Reply:
The tests should not need any special environment. I had the issue that when there was not `/Application/Xcode-12.app` that the unit tests did fail. But I thought I had that fixed. On my box now I only have `/Application/Xcode-12.2.app` and `/Application/Xcode-12.3.app`

I have removed all the `@Ignore` and on my box all unit tests run fine.